NitroxClient/Communication/Packets/Processors/VehicleDockingProcessor.cs

```diff
@@ -36,7 +36,7 @@ public override void Process(VehicleDocking packet)
             vehicle.StartCoroutine(DisablePilotingAfterAnimation(packet.VehicleGuid, packet.PlayerId));
         }
 
-        IEnumerator DisablePilotingAfterAnimation(string vehicleGuid, ushort playerId)
+        IEnumerator DisablePilotingAfterAnimation(string vehicleGuid, ulong playerId)
```

Comment:
The SteamAPI uses a ulong id so it's easier to store this way.

Code that works with SteamID is in JoinServer class:

`AuthenticationContext authenticationContext = new AuthenticationContext(playerName, Steamworks.SteamUser.GetSteamID().m_SteamID);`

Reply:
Hmm, does this break multiple copies running for development purposes?

Reply:
havent tested yet however it shouldnt break it will likely treat all the characters inventories as the same inventory with the duplicate character models going to 0,0,0